src/rsa.rs

```diff
@@ -504,6 +539,123 @@ mod tests {
     #[cfg(feature = "rsa_signing")]
     extern { static GFp_BN_BLINDING_COUNTER: u32; }
 
+    #[cfg(feature = "rsa_signing")]
+    #[test]
+    fn test_signature_rsa_pss_sign() {
+        struct DeterministicSalt<'a> {
```

Comment:
Can you just use `ring::rand::test_util::FixedSliceRandom` here? If not, please explain why in a comment.